From ccbed91b47c3b5c2541f98029a03a7037503c3b2 Mon Sep 17 00:00:00 2001
From: aishwarya_natarajan2 <aishwarya_natarajan2@comcast.com>
Date: Wed, 14 Aug 2024 07:28:24 +0000
Subject: [PATCH] RDKBACCL-275 :  Review generic rdk-b code and introduce
 platform specific flags

Reason for change : Changes for PandM
Test Procedure: Build and flash the image.erouter0 should get ip
Risks: None

Change-Id: I0817da18c5c355b342169e495038e57a58b32b41
Signed-off-by: aishwarya_natarajan2 <aishwarya_natarajan2@comcast.com>
---
 source/dmltad/cosa_selfheal_dml.c | 2 +-
 source/dmltad/diag.c              | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/source/dmltad/cosa_selfheal_dml.c b/source/dmltad/cosa_selfheal_dml.c
index 15d4e79..b60337a 100644
--- a/source/dmltad/cosa_selfheal_dml.c
+++ b/source/dmltad/cosa_selfheal_dml.c
@@ -1592,7 +1592,7 @@ ResourceMonitor_SetParamUlongValue
             return TRUE;
         }
   
-#if defined(_ARRIS_XB6_PRODUCT_REQ_) || defined(_CBR_PRODUCT_REQ_) || defined(_PLATFORM_RASPBERRYPI_) || defined(_PLATFORM_TURRIS_) || \
+#if defined(_ARRIS_XB6_PRODUCT_REQ_) || defined(_CBR_PRODUCT_REQ_) || defined(_PLATFORM_RASPBERRYPI_) || defined(_PLATFORM_TURRIS_) || defined(_PLATFORM_BANANAPI_R4_) || \
 (defined(_XB6_PRODUCT_REQ_) && defined(_COSA_BCM_ARM_))
         char buf[8];
         errno_t rc = -1;
diff --git a/source/dmltad/diag.c b/source/dmltad/diag.c
index f719494..fb0ba38 100644
--- a/source/dmltad/diag.c
+++ b/source/dmltad/diag.c
@@ -496,7 +496,7 @@ static void *diag_task(void *arg)
             break;
     }
 
-#if defined(_PLATFORM_RASPBERRYPI_) || (_PLATFORM_TURRIS_)
+#if defined(_PLATFORM_RASPBERRYPI_) || (_PLATFORM_TURRIS_) || defined(_PLATFORM_BANANAPI_R4_)
 /**
  inet_pton is failing because of extra quotes in ip address (cfg.host)
 **/
-- 
2.47.0

